### ISSUE TITLE
Carthage Support

### ISSUE THREAD

**Issue by jgrund01** [sunshinejr/Moya-ModelMapper#6]
Do you have any plans to add support for [Carthage](https://github.com/Carthage/Carthage)?  Would love to add this nice mapper into an existing project that doesn't have the option to use CocoaPods.

**Comment by sunshinejr** [comment]
Hey @jgrund01, thanks for the issue! Although it seems like Carthage was supported, many libraries have problem with it again so I might need to check it as well. Sorry for the inconvenience!

**Comment by kemchenj** [comment]
Carthage now works fine. May be now is a good timing to add support for Carthage? @sunshinejr

**Comment by sunshinejr** [comment]
Hey @kemchenj, I'd love to but I'm no expert in that field, unfortunately. If someone can help me with that making a PR it would be awesome, if not, I might need to take a deeper look into that.

**Comment by bbqsrc** [comment]
I have made a [pull request](https://github.com/sunshinejr/Moya-ModelMapper/pull/29).

**Comment by sunshinejr** [comment]
Hey guys, I've just merged a PR that could help with Carthage issues. Can anyone of you confirm it works for you? Would be awesome.

**Comment by kemchenj** [comment]
@sunshinejr  I open a new project, run 'carthage update --platform iOS' and got an error like this

```
*** Fetching Moya-ModelMapper
*** Checking out Moya-ModelMapper at "4.1.0"
*** xcodebuild output can be found in /var/folders/p3/5zt2819s47d22d714g8_tsvh0000gn/T/carthage-xcodebuild.9b6SqX.log
*** Skipped building Moya-ModelMapper due to the error:
Dependency "Moya-ModelMapper" has no shared framework schemes

If you believe this to be an error, please file an issue with the maintainers at https://github.com/sunshinejr/Moya-ModelMapper/issues/new
```

macOS 12.3
Xcode 8.3.3

**Comment by sunshinejr** [comment]
@kemchenj I probably need to release a new version for this one, correct?

**Comment by kemchenj** [comment]
Yeah, I think so. I use git path, instead of `github`, and things works fine

```
git "https://github.com/sunshinejr/Moya-ModelMapper.git" "master"
```

**Comment by lswith** [comment]
Once a new version gets release. Carthage will be there. I have updated the readme to include Carthage in https://github.com/sunshinejr/Moya-ModelMapper/pull/34

**Comment by sunshinejr** [comment]
I've released version 4.1.1. It should have Carthage support built-in. Let me know if it's working guys :)

**Comment by sunshinejr** [comment]
This should work in 5.0.0 release as well! Closing this one, but if something happens, please create a new issue!
